src/Form.php

```diff
@@ -529,7 +551,7 @@ public function ajaxSubmit()
         //var_dump($cb->getURL());
         $this->js(true)
             ->api(array_merge(['url' => $cb->getJSURL(), 'method' => 'POST', 'serializeForm' => true], $this->apiConfig))
-            ->form(['inline' => true, 'on' => 'blur']);
+            ->form(array_merge($this->formConfig, ['inline' => true, 'on' => 'blur']));
```

Comment:
it's generally better if the formConfig is second argument and can override the defaults (such as 'on'=>blur) Can you please swap it?